net-mail/vpopmail/vpopmail-5.4.33-r4.ebuild

```diff
@@ -0,0 +1,300 @@
+		eerror "You probably want to check that out."
+		eerror "Continuing with default."
+		VPOP_HOME="${VPOP_DEFAULT_HOME}"
+	else
+		einfo "Setting VPOP_HOME to: $VPOP_HOME"
+	fi
+}
+
+pkg_setup() {
+	enewgroup vpopmail 89
+	enewuser vpopmail 89 -1 ${VPOP_DEFAULT_HOME} vpopmail
+	upgradewarning
+}
+
+src_prepare() {
+	default
+
+	echo 'install-recursive: install-exec-am' \
+		>>"${S}"/Makefile.am
+
+	# fix maildir paths
+	sed -i -e 's|Maildir|.maildir|g' \
+		vchkpw.c vconvert.c vdelivermail.c \
+		vpopbull.c vpopmail.c vqmaillocal.c \
+		vuserinfo.c maildirquota.c || die
+
+	# remove vpopmail advertisement
+	sed -i -e '/printf.*vpopmail/s:vpopmail (:(:' \
+		vdelivermail.c vpopbull.c vqmaillocal.c || die
+
+	# automake/autoconf
+	mv -f "${S}"/configure.{in,ac} || die
+	sed -i -e 's,AM_CONFIG_HEADER,AC_CONFIG_HEADERS,g' \
+	        configure.ac || die
+
+	# _FORTIFY_SOURCE
+	sed -i \
+		-e 's/\(snprintf(\s*\(LI->[a-zA-Z_]\+\),\s*\)[a-zA-Z_]\+,/\1 sizeof(\2),/' \
+		vlistlib.c || die
+
+	eautoreconf
+	ht_fix_file cdb/Makefile
+}
+
+src_configure() {
+	vpopmail_set_homedir
+
+	local authopts
+	if use mysql; then
+		incdir=$(mysql_config --variable=pkgincludedir)
+		libdir=$(mysql_config --variable=pkglibdir)
+		authopts+=" --enable-auth-module=mysql"
+		authopts+=" --enable-incdir=${incdir}"
+		authopts+=" --enable-libdir=${libdir}"
+		authopts+=" --enable-sql-logging"
+		authopts+=" --enable-valias"
+		authopts+=" --disable-mysql-replication"
+		authopts+=" --enable-mysql-limits"
+	elif use postgres; then
+		libdir=$(pg_config --libdir)
+		incdir=$(pg_config --pkgincludedir)
+		authopts+=" --enable-auth-module=pgsql"
+		authopts+=" --enable-incdir=${incdir}"
+		authopts+=" --enable-libdir=${libdir}"
+		authopts+=" --enable-sql-logging"
+		authopts+=" --enable-valias"
+	else
+		authopts+=" --enable-auth-module=cdb"
+	fi
+
+	econf ${authopts} \
+		--sysconfdir=${VPOP_HOME}/etc \
+		--enable-non-root-build \
+		--enable-qmaildir=${QMAIL_HOME} \
+		--enable-qmail-newu=${QMAIL_HOME}/bin/qmail-newu \
+		--enable-qmail-inject=${QMAIL_HOME}/bin/qmail-inject \
+		--enable-qmail-newmrh=${QMAIL_HOME}/bin/qmail-newmrh \
+		--enable-vpopuser=vpopmail \
+		--enable-vpopgroup=vpopmail \
+		--enable-many-domains \
+		--enable-file-locking \
+		--enable-file-sync \
+		--enable-md5-passwords \
+		--enable-logging \
+		--enable-auth-logging \
+		--enable-log-name=vpopmail \
+		--enable-qmail-ext \
+		--disable-tcpserver-file \
+		--disable-roaming-users \
+		$(use_enable ipalias ip-alias-domains) \
+		$(use_enable clearpasswd clear-passwd) \
+		$(use_enable maildrop) \
+		$(use_enable maildrop maildrop-prog /usr/bin/maildrop) \
+		$(use_enable spamassassin)
+}
+
+src_install() {
+	vpopmail_set_homedir
+
+	emake DESTDIR="${D}" install
+	keepdir "${VPOP_HOME}"/domains
+
+	# install helper script for maildir conversion
+	into "${VPOP_HOME}"
+	dobin "${FILESDIR}"/vpopmail-Maildir-dotmaildir-fix.sh
+	into /usr
+
+	einstalldocs
+	rm -rf "${D}/${VPOP_HOME}"/doc
```

Comment:
We generally avoid using `-f` because you ought to know what youre removing ;)

Reply:
legacy, just copied from -r3